website/docs/guides/site_org_association.html.markdown

```diff
@@ -0,0 +1,260 @@
+received yet.
+
+For the association to be completed, the other administrator must perform the same operation, using the XML data file for site1.
+
+```hcl
+# as administrator of site2
+resource "vcd_multisite_site_association" "site2-site1" {
+  association_data_file   = "site1.xml"
+  connection_timeout_mins = 2
+}
+```
+
+### Associating two organizations
+
+To associate your organization with another one, you need to have the association data available, either as an XML file or as
+a text string. If you don't own both organizations, you should receive the data from the other organization administrator, who will
+get the data using [data collection](#data-collection-for-an-organization).
+
+Using the XML file (`org2.xml`) received from the other administrator, you can run the following:
+
+```hcl
+# as administrator of org1
+resource "vcd_multisite_org_association" "org1-org2" {
+  association_data_file   = "org2.xml"
+  connection_timeout_mins = 2
+}
+```
+
+This operation will establish one side of the association between org1 and org2. At this stage, if no other operation
+is performed, the association is in state `ASYMMETRIC`, meaning that it has been initiated, but its counterpart has not been
+received yet.
+
+For the association to be completed, the other administrator must perform the same operation, using the XML data file for org1.
+
+```hcl
+# as administrator of org2
+resource "vcd_multisite_org_association" "org2-org1" {
+  association_data_file   = "org1.xml"
+  connection_timeout_mins = 2
+}
+```
+
+### Checking the association completion
+
+When both sides of the association operation have been performed (in both `vcd_multisite_site_association` or 
+`vcd_multisite_org_association`), you can run `terraform apply` once more. This _update_ operation will use the
+field `connection_timeout_mins` to check the association status. 
+The expression `connection_timeout_mins = 2` means "check for up to
+two minutes whether the status of the connection is `ACTIVE`". If the association reaches the desired status within the
+intended timeout, all is well, and the association is ready to be used. If it fails, it means that either the connection
+at the other side was not executed, or that there is some communication problem.
+
+Note about `connection_timeout_mins`: 
+1. You must not run this check before both sides of the association have run. If you run it with only one side, it will
+  fail, as the status cannot be `ACTIVE`.
+2. The property `connection_timeout_mins` is only evaluated during an _update_. It is safe to have it in the script at
+  creation, as it will be ignored during that operation. 
+
+## Listing associations
+
+### Listing site associations
+
+You can use one of the two methods below:
+
+1. run data source `vcd_resource_list` with `resource_type = "vcd_multisite_site_association"`
+2. run data source `vcd_multisite_site`: it will show the number and the name of site associations.
```

Comment:
Nit (here and same below):

```suggestion
1. Run data source `vcd_resource_list` with `resource_type = "vcd_multisite_site_association"`
2. Run data source `vcd_multisite_site`: it will show the number and the name of site associations.
```

Reply:
Changed